Ports/coreutils/package.sh

```diff
@@ -9,3 +9,11 @@ https://ftpmirror.gnu.org/gnu/coreutils/coreutils-${version}.tar.gz.sig coreutil
 https://ftpmirror.gnu.org/gnu/gnu-keyring.gpg gnu-keyring.gpg"
 auth_type="sig"
 auth_opts=("--keyring" "./gnu-keyring.gpg" "coreutils-${version}.tar.gz.sig")
+
+# Exclude some non-working utilities:
+#  - arch, coreutils, and hostname are the default setting
```

Comment:
Not sure what that means, could you elaborate?

Reply:
coreutils excludes the commands `arch`, `coreutils`, and `hostname` by default when using the default configuration.